chart/Chart.yaml

```
@@ -26,7 +26,7 @@ dependencies:
     repository: https://charts.timescale.com
   - name: promscale
     condition: promscale.enabled
-    version: 13.0.0
+    version: 14.1.1
```

```suggestion
    version: 14.1.2
```

Would be nice if you can do this as part of this PR only.

---

just updated!